### ISSUE TITLE
GOG.com leftovers after games from gog uninstall

### ISSUE THREAD

**Issue by DmitriyYukhanov** [Klocman/Bulk-Crap-Uninstaller#179]
Hey,

I've got Firewatch game from GOG.com.
Installed it, played, uninstalled (lovely game, tho ending was too dolorous).
After uninstallation I've removed leftovers BCU found and noticed it doesn't catch GOG-specific items in registry:

HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\GOG.com
with such items:
![image](https://user-images.githubusercontent.com/883587/36644950-a486db50-1a72-11e8-8794-f4aea2f4dc82.png)

HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\GOG.com\Games
without any items inside (except default one)

HKEY_USERS\S-1-5-21-2081619299-1437623788-1181410189-1001\Software\GOG.com
with such items:
![image](https://user-images.githubusercontent.com/883587/36644959-cf33789a-1a72-11e8-8b39-6005602de27d.png)

I see it's impossible to detect it guessing fromt he uninstalled app name tho it should be relatively possible I guess with registry keys creation dates or maybe with kind of specific rule for GOG games.

Thanks!

**Comment by Klocman** [comment]
Hello!

Maybe GOG. com is still installed? It's an app store like steam, so it might have a separate uninstaller that you didn't notice.

**Comment by DmitriyYukhanov** [comment]
Hey, just double-checked everything and I don't see such app installed, nothing with GOG in the name is installed.

**Comment by Klocman** [comment]
Right, it looks like these are just settings for future GOG installs. It's really hard to link this to the individual games and it has basically no negative effect, so I think I'll leave it be.
